src/test/resources/data/FeedbackSessionsLogicTest.json

```diff
@@ -1329,14 +1329,14 @@
       "commentGiver": "instructor1@course1.tmt",
       "giverSection": "None",
       "receiverSection": "None",
-      "feedbackResponseId": "3%student1InCourse1@gmail.tmt%student1InCourse1@gmail.tmt",
```

Comment:
This `feedbackResponseId` doesn't not exist.
Question 3's response is from `instructor1@course1` to general not `student1` self feedback.
```
    "response1ForQ3S1C1": {
      "feedbackSessionName": "First feedback session",
      "courseId": "idOfTypicalCourse1",
      "feedbackQuestionId": "3",
      "feedbackQuestionType": "TEXT",
      "giver": "instructor1@course1.tmt",
      "recipient": "%GENERAL%",
      "responseMetaData": "Good work, keep it up!",
      "giverSection": "None",
      "recipientSection": "None"
    },
```

The entry is introduced in https://github.com/TEAMMATES/teammates/commit/dfadb18b0acf2d471f0320d121486575cf2cccfb. Probably by mistakes with direct copy.

There are more in the data bundle. Will create a separate issue and fix them in another PR.